src/plugins/intel_gpu/src/kernel_selector/kernels/convolution/convolution_kernel_ref.cpp

```diff
@@ -126,6 +126,12 @@ bool ConvolutionKernel_Ref::Validate(const Params& params) const {
     if (input_type == output_type && input_type != Datatype::UINT8 && input_type != Datatype::INT8)
         return true;
 
+    // For fp model, some convolutions may not be compressed to fp16 depending on the transformation policy,
+    // and those convolutions may have the fused nodes which is of fp16.
+    // Convolution needs to support for this case.
+    if (input_type == Datatype::F32 && output_type == Datatype::F16)
+        return true;
```

Comment:
1. What if fp16 input & f32 output with the same situation? 
2. If it is okay with all combination of fp16 & fp32 or fp32 or fp16, then jist remove the above check of input_type == output_type is enough?
3. or change the line 126 check to 
if (input_type != u8 && input_type != i8 && output_type != u8 && output_type != i8)?

Reply:
The third option seems to cover all cases with minimized modification. Really appreciate.